django-stubs/forms/renderers.pyi

```diff
@@ -9,6 +9,8 @@ from django.template.base import Template
 def get_default_renderer() -> BaseRenderer: ...
 
 class BaseRenderer:
+    form_template_name: str = ...
+    formset_template_name: str = ...
```

Comment:
So these are some of the 4.1 changes. In 4.0 they are on `BaseForm`.

I'm not sure which version is targeted at any given time given the project does not aim to support multiple versions of Django at once?

Reply:
We do not target 4.1 yet.

Reply:
Thanks -- I've rolled back the 4.1 changes. This now targets 4.0.